meta-client/src/test/java/org/apache/helix/metaclient/impl/zk/TestZkMetaClient.java

```diff
@@ -41,8 +53,125 @@ public void prepare() {
   }
 
   @Test
-  public void dummyTest() {
-    Assert.assertEquals(1+1, 2);
+  public void testGet() {
+    final String key = "/TestZkMetaClient_testGet";
+    ZkMetaClient<String> zkMetaClient = createZkMetaClient();
+    String value;
+    zkMetaClient.create(key, ENTRY_STRING_VALUE);
+    String dataValue = zkMetaClient.get(key);
+    Assert.assertEquals(dataValue, ENTRY_STRING_VALUE);
+
+    value = zkMetaClient.get(key + "/a/b/c");
+    Assert.assertNull(value);
+
+    zkMetaClient.delete(key);
+
+    value = zkMetaClient.get(key);
+    Assert.assertNull(value);
```

Comment:
Should we close metaclient to avoid leakage in test? Maybe we can wrap the test in a try-with-resource block.

Reply:
Good point. we should close it in afterClass.